src/components/MenuBar/MenuBarToggleButton.tsx

```
@@ -0,0 +1,50 @@
+ * A toggle button for use in the menu bar.
+ *
+ * @param props
+ * @param props.title Tooltip title.
+ * @param props.tooltipPlacement Tooltip position.
+ * @param props.onIcon Icon when active.
+ * @param props.offIcon Icon when inactive.
+ * @param props.rest Other IconButton props.
+ * @return JSX.Element
+ */
+const MenuBarToggleButton = ({
+                                 tooltipPlacement,
```

ESLint

[error] Expected indentation of 4 spaces but found 33.

(@stylistic/indent)